user/isolated-vs-merged-workspaces.rst

```diff
@@ -0,0 +1,73 @@
+Isolated vs Merged Workspaces
+=============================
+
+Assuming you know :doc:`what a workspace is <what-is-a-workspace>`, you know that by default colcon builds an **isolated workspace**.
+It can also build a **merged workspace**.
+These terms describe the layout of the **install space**, which is the directory software gets installed into.
+
+This page describes the differences between the two.
+
+
+Isolated Workspace
+------------------
+
+Colcon by default builds an isolated workspace.
+
+Build the workspace from :doc:`what-is-a-workspace` without any extra arguments.
+
+.. code-block:: bash
+
+	colcon build
+
+Now let's look in the ``install`` folder.
+
+::
+
+	install
+	├── COLCON_IGNORE
+	├── foo
+	│	├── lib/...
+	│	└── share/...
+	├── local_setup.[bash|bat|ps1|sh|zsh|...]
+	├── _local_setup_util_[sh|ps1|...].py
+	└── setup.[bash|bat|ps1|sh|zsh|...]
+
+Colcon created a directory ``install/foo`` and installed the package ``foo`` inside of it.
+Building an isolated workspace just means every software package is installed into its own directory.
+
+An isolated workspace has some advantages.
+When colcon tests a package in an isolated workspace it will only give the tests access to the install artifacts of the depdencencies it declares.
```

Comment:
```suggestion
When colcon tests a package in an isolated workspace it will only give the tests access to the install artifacts of the dependencies it declares.
```